packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -1,70 +1,163 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+import uniqueid from 'lodash.uniqueid';
 
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
 import RadioField from '../../src/RadioField';
 import Radio from '../../src/Radio';
 
+jest.mock('lodash.uniqueid');
 window.matchMedia = () => ({ matches: true });
 
+let userAgentGetter;
+beforeEach(() => {
+  userAgentGetter = jest.spyOn(window.navigator, 'userAgent', 'get');
+  uniqueid.mockImplementation(() => 'uuid123');
+});
+
 it('should render a default radio field', () => {
-  const radioField = (<RadioField legend="Default RadioField" />);
+  const radioField = <RadioField legend="Default RadioField" />;
   const wrapper = shallowWithIntl(radioField);
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render radio field with div element for Safari browser or Edg browser', () => {
+  userAgentGetter.mockReturnValue('Safari Edg');
+  const wrapper = shallowWithIntl(
+    <RadioField legend="Custom Message RadioField" />,
+  );
+  expect(wrapper.dive()).toMatchInlineSnapshot(`
+    <fieldset
+      className="radio-field"
+      id="terra-radio-group-uuid123"
+      required={false}
+    >
+      <div
+        className="legend-group"
+        id="terra-radio-field-description-uuid123"
+      >
+        <div
+          className="legend"
+        >
+          Custom Message RadioField
+          <span
+            className="error-icon-hidden"
+          />
+        </div>
+      </div>
+    </fieldset>
+  `);
```

Comment:
Is there a reason this is being used instead of `toMatchSnapshot`?

Reply:
As per @supreethmr  suggestion I'm using .toMatchInlineSnapshot().

Reply:
According to the [Jest docs](https://jestjs.io/docs/snapshot-testing#inline-snapshots),

> Inline snapshots behave identically to external snapshots (.snap files), except the snapshot values are written automatically back into the source code.

Since it seems like we'll be able to achieve the same result with either function, let's use `.toMatchSnapshot` for consistency with other Terra components. We can also specifically test for properties (see [`terra-alert`](https://github.com/cerner/terra-core/blob/main/packages/terra-alert/tests/jest/Alert.test.jsx#L29-L40) for an example.)

Reply:
i have made changes as per suggestions.